### PR TITLE
Added logger to the send batch and added request timeout

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: pymess/backend/dialer/daktela.py [hunk 1]
```diff
@@ -30,7 +30,11 @@ def _update_dialer_states(self, messages):
         for message in messages:
             name = message.extra_data['name']
             client_url = self._get_dialer_api_url(name)
-            response = generate_session(slug=self.SESSION_SLUG, related_objects=(message,)).get(client_url)
+            response = generate_session(
+                slug=self.SESSION_SLUG,
+                related_objects=(message,),
+                timeout=self.DIALER_DAKTELA.TIMEOUT
```
Comment (matllubos) [inline pymess/backend/dialer/daktela.py]: default timeout

File: pymess/management/commands/send_emails_batch.py [hunk 2]
```diff
@@ -23,8 +28,10 @@ def handle(self, *args, **options):
             self.stdout.write('acquiring lock...')
             lock.acquire(settings.EMAIL_BATCH_LOCK_WAIT_TIMEOUT)
         except AlreadyLocked:
+            logger.error('Send emails batch: lock already in place.')
             raise CommandError('lock already in place. quitting.')
```
Comment (matllubos) [inline pymess/management/commands/send_emails_batch.py]: CommandError is not logged by default :(

File: pymess/utils/logged_requests.py [hunk 3]
```diff
@@ -1,9 +1,26 @@
-def generate_session(slug=None, related_objects=None):
+class DefaultTimeoutSessionMixin:
```
Comment (matllubos) [inline pymess/utils/logged_requests.py]: I don't understand why requests doesn't have this settings by default. Timeout is often forgotten

File: pymess/utils/logged_requests.py [hunk 4]
```diff
@@ -1,9 +1,26 @@
...
+        timeout = kwargs.pop('timeout', self.timeout)
+        return super().request(method, url, timeout=timeout, **kwargs)
+
+
+def generate_session(slug=None, related_objects=None, timeout=None):
     try:
         from security.transport.security_requests import SecuritySession
 
-        return SecuritySession(slug=slug, related_objects=related_objects)
+        class DefaultTimeoutSecuritySession(DefaultTimeoutSessionMixin, SecuritySession):
```
Comment (Formulka) [inline pymess/utils/logged_requests.py]: why are these defined inside the function?

Comment (matllubos) [reply pymess/utils/logged_requests.py]: because the funcition generates classes according to installed libraries

**Pull request merged**